csv2geojson

```diff
@@ -2,7 +2,7 @@
 
 var optimist = require('optimist'),
     fs = require('fs'),
-    sys = require('sys'),
+    sys = require('util'),
```

Comment:
please rename the variable as well, and references to it, to avoid the technical debt of a misnamed require